core/server/master/src/test/java/alluxio/master/AlluxioMasterRestServiceHandlerTest.java

```diff
@@ -91,9 +90,15 @@
   private BlockMaster mBlockMaster;
   private MasterRegistry mRegistry;
   private AlluxioMasterRestServiceHandler mHandler;
+
   @Rule
   public TemporaryFolder mTestFolder = new TemporaryFolder();
 
+  @Rule
+  public ConfigurationRule mConfigurationRule = new ConfigurationRule(new HashMap() {{
+      put(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS, TEST_PATH);
+  }});
```

Comment:
The builder one is unnecessarily verbose in general. It's easier to read these rules when the first line doesn't wrap.